packages/react-aria-components/src/Breadcrumbs.tsx

```diff
@@ -9,16 +9,39 @@
  * OF ANY KIND, either express or implied. See the License for the specific language
  * governing permissions and limitations under the License.
  */
-import {AriaBreadcrumbsProps, useBreadcrumbs} from 'react-aria';
+import {AriaBreadcrumbsProps, mergeProps, useBreadcrumbs, useFocusRing, useHover} from 'react-aria';
 import {CollectionProps, useCollection} from './Collection';
-import {ContextValue, forwardRefType, Provider, SlotProps, StyleProps, useContextProps} from './utils';
+import {ContextValue, forwardRefType, Provider, RenderProps, SlotProps, useContextProps, useRenderProps} from './utils';
 import {filterDOMProps} from '@react-aria/utils';
 import {HeadingContext} from './Heading';
 import {LinkContext} from './Link';
 import {Node} from 'react-stately';
 import React, {createContext, ForwardedRef, forwardRef, HTMLAttributes} from 'react';
 
-export interface BreadcrumbsProps<T> extends Omit<CollectionProps<T>, 'disabledKeys'>, Omit<AriaBreadcrumbsProps, 'children'>, StyleProps, SlotProps {
+export interface BreadcrumbsRenderProps {
+  /**
+   * Whether the breadcrumbs are currently hovered with a mouse.
+   * @selector [data-hovered]
+   */
+  isHovered: boolean,
+  /**
+   * Whether the breadcrumbs are disabled.
+   * @selector [data-disabled]
+   */
+  isDisabled: boolean,
+  /**
+   * Whether an element within the breadcrumbs is focused, either via a mouse or keyboard.
+   * @selector :focus-within
+   */
+  isFocusWithin: boolean,
+  /**
+   * Whether an element within the breadcrumbs is keyboard focused.
+   * @selector [data-focus-visible]
+   */
+  isFocusVisible: boolean
+}
+
+export interface BreadcrumbsProps<T> extends Omit<CollectionProps<T>, 'disabledKeys' | 'children'>, Omit<AriaBreadcrumbsProps, 'children'>, SlotProps, RenderProps<BreadcrumbsRenderProps> {
```

Comment:
Should be the following right? 
```suggestion
export interface BreadcrumbsProps<T> extends Omit<CollectionProps<T>, 'disabledKeys'>, Omit<AriaBreadcrumbsProps, 'children'>, SlotProps, RenderProps<BreadcrumbsRenderProps> {
```

Reply:
I think we want to omit `children` since it isn't compatible with `children` in `RenderProps`, and that is one preferred.